File: ejercicicos con jose cuellar/Python/ejercicio9.py
# Escribe un programa que pida al usuario que introduzca los segundos, y le conteste diciéndole el número de días, horas, minutos y segundos que son.

day = 86400
hour = 3600
minute = 60

segundo = int(input("Ingrese la cantidad de segundos: "))

dia = int(segundo/day)
hora = int(segundo/hour)
minuto = int(segundo/minute) 

print('La cantidad de dias son',dia,'\n La cantidad de horas son',hora,'\n La cantidad de minutos son',minuto,'\n La cantidad de segundos son',segundo)

File: ejercicicos con jose cuellar/js/js/ejercicio9.js
function calcular(){

    const dia = 86400;
    const hora = 3600;
    const minuto = 60;

    let segundo

    segundo = document.getElementById("segundo").value

    day = segundo/dia;
    hour = segundo/hora;
    minutes = segundo/minuto;

    print = document.getElementById("respuesta")
    print.innerHTML = "La cantidad de dias son: " + day + "<br> La cantidad de horas son: " + hour + "<br> La cantidad de minutos son: " + minutes + "<br> La cantidad de segundos son: " + segundo;
}